scripts/vote_2021_09_02.py

```diff
@@ -0,0 +1,255 @@
+"""
+Voting 02/09/2021.
+
+1. Node Operators Registry:
+   Add node operator named Blockdaemon with reward address
+   `0x4f42A816dC2DBa82fF927b6996c14a741DCbD902`
+2. Raise key limit for Node Operator #2 (p2p) to 4800
+3. Raise key limit for Node Operator #4 (stakefish) to 6000
+4. Raise key limit for Node Operator #5 (Blockscape) to 7000
+5. Raise key limit for Node Operator #6 (DSRV) to 3700
+6. Raise key limit for Node Operator #8 (SkillZ) to 7000
+7. Raise key limit for Node Operator #9 (RockX) to 100
+8. Raise key limit for Node Operator #10 (Figment) to 100
+9. Raise key limit for Node Operator #11 (Allnodes) to 100
+10. Raise key limit for Node Operator #12 (Anyblock) to 100
+11. Allocate LDO tokens (3 792 328.15 LDO) for referral rewards
+    (to DeversiFi and Ledger) to finance
+    address `0x48F300bD3C52c7dA6aAbDE4B683dEB27d38B9ABb`
+"""
+import time
+from functools import partial
+from typing import (
+    Dict, Tuple,
+    Optional
+)
+
+from brownie.utils import color
+from brownie.network.transaction import TransactionReceipt
+
+from utils.voting import create_vote
+from utils.finance import encode_token_transfer
+from utils.node_operators import (
+    encode_add_operator,
+    encode_set_node_operator_staking_limit
+)
+from utils.evm_script import (
+    decode_evm_script,
+    encode_call_script,
+    calls_info_pretty_print
+)
+from utils.config import (
+    prompt_bool,
+    get_deployer_account,
+    ldo_token_address,
+    lido_dao_voting_address,
+    lido_dao_finance_address,
+    lido_dao_token_manager_address,
+    lido_dao_node_operators_registry,
+)
+
+try:
+    from brownie import interface
+except ImportError:
+    print(
+        'You\'re probably running inside Brownie console. '
+        'Please call:\n'
+        'set_console_globals(interface=interface)'
+    )
+
+
+def set_console_globals(**kwargs):
+    """Extract interface from brownie environment."""
+    global interface
+    interface = kwargs['interface']
+
+
+def pp(text, value):
+    """Pretty print with colorized."""
+    print(text, color.highlight(str(value)), end='')
+
+
+def as_wei(amount: int, n_decimals: int = 0) -> int:
+    """Convert amount to wei."""
+    factor = 10 ** (18 - n_decimals)
+    return amount * factor
+
+
+def make_ldo_referral_payout(
+        *not_specified,
+        target_address: str, ldo_in_wei: int,
+        finance: interface.Finance
+) -> Tuple[str, str]:
+    """Encode referral payout for DeversiFi and Ledger"""
+    if not_specified:
+        raise ValueError(
+            'Please, specify all arguments with keywords.'
+        )
+
+    return encode_token_transfer(
+        token_address=ldo_token_address,
+        recipient=target_address,
+        amount=ldo_in_wei,
+        reference=f'Referral payout for DeversiFi and Ledger to: '
```

Comment:
Let's stick to "Refferal program third period payout"

Reply:
Done